### ISSUE TITLE
OpenLab Homepage: Featured Slider might have incorrect ARIA Role

### ISSUE THREAD

**Issue by beckej13820** [cuny-academic-commons/commons-in-a-box#383]
I've been looking at the OpenLab theme with an accessibility checker called [Axe Dev Tools](https://www.deque.com/axe/devtools/).

On the OpenLab HomePage, we get the error "ARIA roles used must conform to valid values." Refering to the `<div class="camera_wrap_sr" role="widget">
`
Axe says, "To solve this issue you need to.... Fix the following: Abstract role cannot be directly used: Widget"

From their [documentation](https://dequeuniversity.com/rules/axe/4.4/aria-roles?application=AxeChrome
), Widget is a "type" of aria role, but the specific role would be more appropriate to label that as a "slider" 

Requirement(s)
WCAG 2.1 (A): MUST
WCAG 2.0 (A): MUST
WCAG Success Criteria
4.1.2 Name, Role, Value

**Comment by boonebgorges** [comment]
Yes, 'widget' is not correct. `slider` is not correct either - it refers to a slideable UI (like a slider for selecting a number within a range). `listbox` looks correct. See https://stackoverflow.com/questions/16840054/wai-aria-roles-for-carousels-a-k-a-sliders-slideshows-galleries, https://developer.mozilla.org/en-US/docs/web/accessibility/aria/roles/listbox_role But it's not really possible to make `listbox` work on this specific element because of the structure of the child markup and the JS that depends on it. As such, I'm just going to remove the incorrect ARIA role altogether.

**Comment by beckej13820** [comment]
Thanks Boone!
